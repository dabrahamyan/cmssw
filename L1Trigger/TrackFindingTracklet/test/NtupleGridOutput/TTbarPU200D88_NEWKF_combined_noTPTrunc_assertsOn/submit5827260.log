000 (5827260.000.000) 09/03 13:21:24 Job submitted from host: <137.138.44.55:9618?addrs=137.138.44.55-9618+[2001-1458-d00-1a--48]-9618&alias=bigbird17.cern.ch&noUDP&sock=schedd_7840_2a97>
...
022 (5827260.000.000) 09/03 13:22:38 Job disconnected, attempting to reconnect
    Socket between submit and execute hosts closed unexpectedly
    Trying to reconnect to slot1_40@b7s01p9025.cern.ch <128.142.56.108:9618?addrs=128.142.56.108-9618+[2001-1458-301-6--100-6b]-9618&alias=b7s01p9025.cern.ch&noUDP&sock=startd_2865_baec>
...
024 (5827260.000.000) 09/03 13:22:38 Job reconnection failed
    Job not found at execution machine
    Can not reconnect to slot1_40@b7s01p9025.cern.ch, rescheduling job
...
022 (5827260.000.000) 09/03 13:32:30 Job disconnected, attempting to reconnect
    Socket between submit and execute hosts closed unexpectedly
    Trying to reconnect to slot1_13@b7g12p9477.cern.ch <128.142.132.131:9618?addrs=128.142.132.131-9618+[2001-1458-301-5a--100-77]-9618&alias=b7g12p9477.cern.ch&noUDP&sock=startd_2960_e46c>
...
024 (5827260.000.000) 09/03 13:32:30 Job reconnection failed
    Job not found at execution machine
    Can not reconnect to slot1_13@b7g12p9477.cern.ch, rescheduling job
...
022 (5827260.000.000) 09/03 13:35:26 Job disconnected, attempting to reconnect
    Socket between submit and execute hosts closed unexpectedly
    Trying to reconnect to slot1_15@b7s11p8141.cern.ch <188.184.162.224:9618?addrs=188.184.162.224-9618&alias=b7s11p8141.cern.ch&noUDP&sock=startd_3339_dad0>
...
024 (5827260.000.000) 09/03 13:35:26 Job reconnection failed
    Job not found at execution machine
    Can not reconnect to slot1_15@b7s11p8141.cern.ch, rescheduling job
...
040 (5827260.000.000) 09/03 13:36:26 Started transferring input files
	Transferring to host: <188.184.162.210:9618?addrs=188.184.162.210-9618+[2001-1458-301-4a--100-cc]-9618&alias=b7s11p9993.cern.ch&noUDP&sock=slot1_12_132335_ba9e_185170>
...
040 (5827260.000.000) 09/03 13:36:26 Finished transferring input files
...
001 (5827260.000.000) 09/03 13:36:27 Job executing on host: <188.184.162.210:9618?addrs=188.184.162.210-9618+[2001-1458-301-4a--100-cc]-9618&alias=b7s11p9993.cern.ch&noUDP&sock=startd_101356_5d32>
...
006 (5827260.000.000) 09/03 13:36:36 Image size of job updated: 280312
	121  -  MemoryUsage of job (MB)
	123456  -  ResidentSetSize of job (KB)
...
006 (5827260.000.000) 09/03 13:38:36 Image size of job updated: 2452124
	121  -  MemoryUsage of job (MB)
	123456  -  ResidentSetSize of job (KB)
...
040 (5827260.000.000) 09/03 13:38:36 Started transferring output files
...
040 (5827260.000.000) 09/03 13:38:36 Finished transferring output files
...
005 (5827260.000.000) 09/03 13:38:36 Job terminated.
	(1) Normal termination (return value 134)
		Usr 0 00:01:38, Sys 0 00:00:09  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:01:38, Sys 0 00:00:09  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	21149  -  Run Bytes Sent By Job
	698  -  Run Bytes Received By Job
	21149  -  Total Bytes Sent By Job
	698  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1         1 
	   Disk (KB)            :       36        1   2832797 
	   Memory (MB)          :      121     2000      2000 

	Job terminated of its own accord at 2023-09-03T11:38:36Z.
...
