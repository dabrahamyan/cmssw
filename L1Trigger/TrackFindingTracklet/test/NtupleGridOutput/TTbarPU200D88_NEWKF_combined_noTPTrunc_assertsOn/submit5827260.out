using geometry D88 (tilted)
Sun Sep  3 13:38:19 CEST 2023
Thread 7 (Thread 0x2b4c6195b700 (LWP 184) "cmsRun"):
#0  0x00002b4c2ddf3b3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002b4c2ddf3bcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002b4c2ddf3c6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002b4c342d0576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002b4c342d0639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002b4c2ddedea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002b4c2e100b0d in clone () from /lib64/libc.so.6
Thread 6 (Thread 0x2b4c6175a700 (LWP 183) "cmsRun"):
#0  0x00002b4c2ddf3b3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002b4c2ddf3bcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002b4c2ddf3c6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002b4c342d0576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002b4c342d0639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002b4c2ddedea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002b4c2e100b0d in clone () from /lib64/libc.so.6
Thread 5 (Thread 0x2b4c61559700 (LWP 182) "cmsRun"):
#0  0x00002b4c2ddf3b3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002b4c2ddf3bcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002b4c2ddf3c6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002b4c342d0576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002b4c342d0639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002b4c2ddedea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002b4c2e100b0d in clone () from /lib64/libc.so.6
Thread 4 (Thread 0x2b4c61358700 (LWP 181) "cmsRun"):
#0  0x00002b4c2ddf4e9d in nanosleep () from /lib64/libpthread.so.0
#1  0x00002b4c3400d10f in XrdSysTimer::Wait(int) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#2  0x00002b4c3423e2d3 in XrdCl::TaskManager::RunTasks() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#3  0x00002b4c3423e429 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002b4c2ddedea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002b4c2e100b0d in clone () from /lib64/libc.so.6
Thread 3 (Thread 0x2b4c61157700 (LWP 180) "cmsRun"):
#0  0x00002b4c2e1010e3 in epoll_wait () from /lib64/libc.so.6
#1  0x00002b4c340075c2 in XrdSys::IOEvents::PollE::Begin(XrdSysSemaphore*, int&, char const**) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#2  0x00002b4c34003d4d in XrdSys::IOEvents::BootStrap::Start(void*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#3  0x00002b4c3400c818 in XrdSysThread_Xeq () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#4  0x00002b4c2ddedea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002b4c2e100b0d in clone () from /lib64/libc.so.6
Thread 2 (Thread 0x2b4c518ae700 (LWP 173) "cmsRun"):
#0  0x00002b4c2ddf51d9 in waitpid () from /lib64/libpthread.so.0
#1  0x00002b4c348a1ad7 in edm::service::cmssw_stacktrace_fork() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#2  0x00002b4c348a26ca in edm::service::InitRootHandlers::stacktraceHelperThread() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#3  0x00002b4c2d9c9f90 in std::execute_native_thread_routine (__p=0x2b4c3cf02060) at ../../../../../libstdc++-v3/src/c++11/thread.cc:80
#4  0x00002b4c2ddedea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002b4c2e100b0d in clone () from /lib64/libc.so.6
Thread 1 (Thread 0x2b4c30183e40 (LWP 155) "cmsRun"):
#0  0x00002b4c2e0f5ddd in poll () from /lib64/libc.so.6
#1  0x00002b4c348a1dcf in full_read.constprop () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#2  0x00002b4c348a279c in edm::service::InitRootHandlers::stacktraceFromThread() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#3  0x00002b4c348a524b in sig_dostack_then_abort () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#4  <signal handler called>
#5  0x00002b4c2e038387 in raise () from /lib64/libc.so.6
#6  0x00002b4c2e039a78 in abort () from /lib64/libc.so.6
#7  0x00002b4c2e0311a6 in __assert_fail_base () from /lib64/libc.so.6
#8  0x00002b4c2e031252 in __assert_fail () from /lib64/libc.so.6
#9  0x00002b4c5ed6e331 in trklet::TrackletProcessor::execute(unsigned int, double, double) () from /afs/cern.ch/user/d/dabraham/private/test/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libL1TriggerTrackFindingTracklet.so
#10 0x00002b4c5eceab59 in trklet::Sector::executeTP() () from /afs/cern.ch/user/d/dabraham/private/test/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libL1TriggerTrackFindingTracklet.so
#11 0x00002b4c5ed5b263 in trklet::TrackletEventProcessor::event(trklet::SLHCEvent&, std::vector<std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > >, std::allocator<std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > > >&, std::vector<std::vector<trklet::StubStreamData, std::allocator<trklet::StubStreamData> >, std::allocator<std::vector<trklet::StubStreamData, std::allocator<trklet::StubStreamData> > > >&) () from /afs/cern.ch/user/d/dabraham/private/test/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libL1TriggerTrackFindingTracklet.so
#12 0x00002b4c5eb95a74 in L1FPGATrackProducer::produce(edm::Event&, edm::EventSetup const&) () from /afs/cern.ch/user/d/dabraham/private/test/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginTrackFindingTrackletPlugins.so
#13 0x00002b4c2ba109ae in edm::one::EDProducerBase::doEvent(edm::EventTransitionInfo const&, edm::ActivityRegistry*, edm::ModuleCallingContext const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#14 0x00002b4c2b9fa23f in edm::WorkerT<edm::one::EDProducerBase>::implDo(edm::EventTransitionInfo const&, edm::ModuleCallingContext const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#15 0x00002b4c2b94ab25 in decltype ({parm#1}()) edm::convertException::wrap<edm::Worker::runModule<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >(edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::TransitionInfoType const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::Context const*)::{lambda()#1}>(edm::Worker::runModule<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >(edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::TransitionInfoType const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::Context const*)::{lambda()#1}) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#16 0x00002b4c2b94ae13 in std::__exception_ptr::exception_ptr edm::Worker::runModuleAfterAsyncPrefetch<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >(std::__exception_ptr::exception_ptr, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::TransitionInfoType const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::Context const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#17 0x00002b4c2b94db5a in void edm::SerialTaskQueueChain::actionToRun<edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >::execute()::{lambda()#1}&>(edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >::execute()::{lambda()#1}&) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#18 0x00002b4c2b94ddd1 in edm::SerialTaskQueue::QueuedTask<edm::SerialTaskQueueChain::push<edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >::execute()::{lambda()#1}&>(tbb::detail::d1::task_group&, edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >::execute()::{lambda()#1}&)::{lambda()#1}>::execute() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#19 0x00002b4c2b6e5175 in tbb::detail::d1::function_task<edm::SerialTaskQueue::spawn(edm::SerialTaskQueue::TaskBase&)::{lambda()#1}>::execute(tbb::detail::d1::execution_data&) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreConcurrency.so
#20 0x00002b4c2d1e0bec in tbb::detail::r1::task_dispatcher::local_wait_for_all<false, tbb::detail::r1::external_waiter> (waiter=..., t=0x2b4c31895f00, this=0x2b4c3188be00) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/task_dispatcher.h:322
#21 tbb::detail::r1::task_dispatcher::local_wait_for_all<tbb::detail::r1::external_waiter> (waiter=..., t=<optimized out>, this=0x2b4c3188be00) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/task_dispatcher.h:463
#22 tbb::detail::r1::task_dispatcher::execute_and_wait (t=<optimized out>, wait_ctx=..., w_ctx=...) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/task_dispatcher.cpp:168
#23 0x00002b4c2b8c522d in edm::FinalWaitingTask::wait() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#24 0x00002b4c2b8ad310 in edm::EventProcessor::processRuns() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#25 0x00002b4c2b8b9da1 in edm::EventProcessor::runToCompletion() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#26 0x000000000040a266 in tbb::detail::d1::task_arena_function<main::{lambda()#1}::operator()() const::{lambda()#1}, void>::operator()() const ()
#27 0x00002b4c2d1cf0eb in tbb::detail::r1::task_arena_impl::execute (ta=..., d=...) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/arena.cpp:698
#28 0x000000000040b094 in main::{lambda()#1}::operator()() const ()
#29 0x000000000040971c in main ()
